cxx/python/mspass_wrapper.cpp

```diff
@@ -437,10 +437,11 @@ PYBIND11_MODULE(mspasspy,m)
   );
   py::enum_<mspass::MDDefFormat>(m,"MDDefFormat")
     .value("PF",MDDefFormat::PF)
-    .value("SimpleText",MDDefFormat::SimpleText)
+    .value("YAML",MDDefFormat::YAML)
   ;
   py::class_<mspass::MetadataDefinitions>(m,"MetadataDefinitions","Load a catalog of valid metadata names with types defined")
     .def(py::init<>())
```

Comment:
Is this default constructor already commented out in the above header `MetadataDefinitions.h`?

Reply:
Yes, good point but I do plan to restore the default constructor.   The goal is to have the default constructor automatically load mspass